.circleci/config.yml

```diff
@@ -6,13 +6,61 @@ executors:
     environment:
       GIT_ORG: astronomer
     docker:
-      - image: circleci/python:3
+      - image: cimg/python:3.10
+
+orbs:
+  snyk: snyk/snyk@1.1.2
+  trivy: signavio/trivy@1.3.0
+
+parameters:
+  scan-docker-images:
+    type: boolean
+    default: false
+    description: "This will run docker image scan."
+
 jobs:
+  snyk-scan-docker:
+    executor: docker-executor
+    parameters:
+      docker_image:
+        type: string
+    steps:
+      - setup_remote_docker:
+          version: 20.10.12
+      - checkout
+      - run:
+          name: Pull Docker image
+          command: docker pull << parameters.docker_image >>
+      - snyk-docker-test:
+          docker_image: << parameters.docker_image >>
+  trivy-scan-docker:
+    executor: docker-executor
+    parameters:
+      docker_image:
+        type: string
+    steps:
+      - setup_remote_docker:
+          version: 20.10.12
+      - checkout
+      - run:
+          name: Pull Docker image
+          command: docker pull << parameters.docker_image >>
+      - trivy-docker-test:
+          docker_image: << parameters.docker_image >>
```

Comment:
It may be cleaner to do this with a matrix. Even though it's auto-generated by python, a matrix could potentially make it more legible.

EG: https://github.com/astronomer/ci-images/blob/main/.circleci/config.yml

Reply:
Done